docs/topics/appendix-proficiency-levels.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[proficiency_levels]]
+= Proficiency Levels
+
+Using each mission available on {launcher} requires certain minimum knowledge from you, and provides a certain information to you. For easier understanding, the requirements and concepts are organized in several levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
```

Comment:
Consider revising:


Each mission teaches you about certain topics, but you need a certain amount of minimum knowledge in order to complete them, which varies by mission. For clarity, the minimum requirements and concepts required by a mission are organized in several proficiency levels. In addition to the levels described in this chapter, there may be additional requirements with each mission, specific to its aim or the technologies it uses.